public/info/works/under-construction.md

```diff
@@ -0,0 +1 @@
+# Coming Soon.
```

Comment:
Are you sure we don't want any kind of styling here?

Reply:
yes boss